src/watcher/watcher.worker.ts

```diff
@@ -67,6 +89,46 @@ class Parser {
         this.queue.push(...paths);
         if (!this.parsing) this.parse();
     }
+    processContent(path: string, content: string) {
+        if (this.debug)
+            console.debug(`Process Content: ${path}`)
+        let fileDetails = this.pendingContentProcessing.get(path)
+        let statBlock = this.findFirstStatBlock(content);
+        if (statBlock) {
+            if (this.debug)
+                console.debug(`StatBlocks: ${JSON.stringify(statBlock)}`)
+            const monster: Monster = Object.assign({}, YAML.parse(statBlock), {
```

Comment:
You should use Obsidian's native `parseYaml` method: https://github.com/obsidianmd/obsidian-api/blob/902badd38ba907689f0917d7b193f7e33d1284fe/obsidian.d.ts#L2540

Reply:
I tried that, but it does'n't seem to be available to the Worker.  If you think it should be, I'll try again.

Reply:
You're right, it isn't... good call